core/src/main/java/com/appintro/core/indicator/DotIndicatorController.kt

```diff
@@ -8,20 +8,19 @@ import android.widget.ImageView
 import android.widget.LinearLayout
 import androidx.core.content.ContextCompat
 import androidx.core.graphics.drawable.DrawableCompat
-import com.github.appintro.R
 
 /**
  * An [IndicatorController] that shows a list of dots and highlight the selected dot.
  * Use this when the number of page you're dealing with is not too high.
  */
 class DotIndicatorController(context: Context) : IndicatorController, LinearLayout(context) {
```

Comment:
This `IndicatorController` implementation depends on Views. It should be more suitable for the view-specific module